_sessions/acrichto.html

```diff
@@ -0,0 +1,23 @@
+---
+layout: session
+permalink: /sessions/:name
+type: talk
+
+by: acrichto
+speaker: Alex Crichton
+title:
```

Comment:
I'm just realizing we're missing the title here. 😶

Reply:
@alexcrichton if you, by any chance, could fill this in, I would announce this around 8AM CEST

Reply:
Oops sorry! Done now

Reply:
Thanks a bunch! This will go live in an hour. ;)